locale/pt-br/blog/uncategorized/an-easy-way-to-build-scalable-network-programs.md

```diff
@@ -0,0 +1,19 @@
+---
+title: An Easy Way to Build Scalable Network Programs
```

Comment:
I think the `title` values also need to get translated

Reply:
lol my bad, will do